pkg/api/graph/graphview/veneering_test.go

```diff
@@ -316,7 +320,7 @@ func TestGraph(t *testing.T) {
 		}
 	}
 
-	if dc != 2 || bc != 3 || ist != 3 || other != 12 {
+	if dc != 2 || bc != 3 || ist != 3 || other != 15 {
```

Comment:
> I can see why 12 wasn't working well. 15 is definitely better

Removed it.  Since we're explicitly creating the nodes now, it's not very interesting.